make broken

Hi, build broken cause go problems

### Current Behavior
output: mod/config.go:23:4: this value of t is never used (SA4006)
pkg/template/template.go:52:17: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead.  (SA1019)
make: *** [Makefile:37 : lint-go] Erreur 1

### Expected Behavior
build successful

### Steps To Reproduce
1. git clone
2. cd regclient
3. make

--- a/docs/README.md
+++ b/docs/README.md
@@ -59,8 +59,6 @@ The following functions have been added in addition to the defaults available in
     Returns current time object, e.g. `{{ $t := time.Now }}{{printf "%d%d%d" $t.Year $t.Month $t.Day}}`.
   - `time.Parse`:
     Parses string using layout into time object, e.g. `{{ $t := time.Parse "2006-01-02" "2020-06-07"}}`.
-- `title`:
-  Makes the first letter of each word uppercase.
 - `upper`:
   Converts a string to uppercase.
 

--- a/pkg/template/template.go
+++ b/pkg/template/template.go
@@ -49,7 +49,6 @@ var tmplFuncs = gotemplate.FuncMap{
 	"lower":       strings.ToLower,
 	"split":       strings.Split,
 	"time":        func() *TimeFuncs { return &TimeFuncs{} },
-	"title":       strings.Title,
 	"upper":       strings.ToUpper,
 }
 

